src/java/org/apache/cassandra/service/accord/AccordSafeCommand.java

```diff
@@ -27,9 +27,33 @@
 import accord.local.Listeners;
 import accord.local.SafeCommand;
 import accord.primitives.TxnId;
+import org.apache.cassandra.utils.concurrent.Ref;
 
 public class AccordSafeCommand extends SafeCommand implements AccordSafeState<TxnId, Command>
 {
+    public static class DebugAccordSafeCommand extends AccordSafeCommand
```

Comment:
I think on balance it would probably be helpful to be able to use these for validation testing with a certain paranoia or debug setting. By itself this wasn't enough to debug the problem (the Ref changes and peppering log statements that would be printed by the leaked references was what tracked the issue down), but it was enough to tell that we had a problem, and that we needed to do that. I'll add a suitable flag.